<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-GK61F68" ignored="0" name="Default test" tests="1" failures="1" timestamp="2020-12-18T19:30:11 IST" time="56.546" errors="0">
  <testcase name="runScenario" time="37.406" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [Testyantra] but found [Testyantra871]">
      <![CDATA[java.lang.AssertionError: expected [Testyantra] but found [Testyantra871]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at stepDefinitions.OrganizationStepDef.i_navigate_to_organization_information_and_verify(OrganizationStepDef.java:49)
at âœ½.I navigate to organization information and verify(file:///D:/BDD/SDET8BDD/./src/test/resources/Features/Organization.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
